variables.tf

```diff
@@ -197,6 +197,18 @@ variable "dapr_component_secrets" {
   sensitive   = true
 }
 
+variable "env_storage" {
+  type = set(object({
```

Comment:
It's better to use `map` here since this variable is used as iterator for `for_each`.